### PR TITLE
solc 0.7 compatibility

### PR DESCRIPTION
**Pull request by xwvvvvwx**

### CODE REVIEW

**Comment by MrChico** [comment]
The changes themselves are fine, are you prepared to recursively update all of dappsys?

**Comment by xwvvvvwx** [comment]
can't wait 😊

**Review by MrChico — APPROVED** [review]

File: src/test.sol [hunk 1]
```diff
@@ -28,7 +28,7 @@ contract DSTest {
     bool public IS_TEST;
     bool public failed;
 
-    constructor() internal {
+    constructor() {
```
Comment (rainbreak) [inline src/test.sol:31]: I think it needs to be explicitly `public` if we're still to be compatible with whatever made us put `internal` there in the first place.

Comment (rainbreak) [reply src/test.sol:31]: I'm making a bigger pr with some other additions, so can address it there. Also have a script for recursive dappsys updating :)

**Comment by rainbreak** [comment]
#16

**Pull request closed without merging**